src/subclass/list_box.rs

```diff
@@ -0,0 +1,208 @@
+
+use glib::translate::*;
+
+use glib::subclass::prelude::*;
+
+use super::container::ContainerImpl;
+use super::widget::WidgetImpl;
+use ListBoxClass;
+use WidgetClass;
+use ListBox;
+use ListBoxRow;
+
+pub trait ListBoxImpl: ListBoxImplExt + ContainerImpl + WidgetImpl {
+    fn row_activated(&self, list_box: &ListBox, row: &ListBoxRow) {
+        self.list_box_row_activated(list_box, row)
+    }
+
+    fn row_selected(&self, list_box: &ListBox, row: &ListBoxRow) {
```

Comment:
```suggestion
    fn row_selected(&self, list_box: &ListBox, row: Option<&ListBoxRow>) {
```

It can be `None` according to the docs